docs/changes.rst

```diff
@@ -1,6 +1,28 @@
 Geofront Changelog
 ==================
 
+Version 0.5.0
+-------------
+
+Released on XX XX, 2017.
+
+- Geofront now supports access to servers inside remote private networks
+  via SSH tunneling over an HTTP WebSocket.  [:issue:`21`]
+
+  - Added ``addresser`` and ``filter`` options to :class:`CloudRemoteSet
+    <geofront.backends.cloud.CloudRemoteSet>` constructor, which allow
+    customization of server addressing and filtering schemes.  The default
+    is still to use public IPs, so you would wish to give lambda functions that
+    return private IPs from the :class:`libcloud.compute.base.Node` object
+    for use with SSH tunneling.
+  - Removed ``--trusted-proxy`` option from the :program:`geofront-server`
+    command.
```

Comment:
These two changes seem to have no direct relationship to SSH tunneling.  Shouldn't these two bullet points be dedented?